lib/perl/Genome/Model/ClinSeq/Command/Converge/Base.pm

```diff
@@ -483,7 +483,7 @@ sub getModelsBuilds{
     my @models = $mg->models;
     $target_count = scalar(@models);
   }
-  if ($verbose){Genome::Model->debug_message("\nSearching for $target_count models/builds");}
+  if ($verbose){$self->debug_message("\nSearching for $target_count models/builds");}
```

Comment:
ok, i removed a bunch here https://github.com/gatoravi/genome/commit/e6e508d6af775554f956d499642e6cc2d5f5f744, I'll work on removing the rest here. There might be a lot of files in ClinSeq with this issue, that might have to be a separate PR I think.